job-templates/kubernetes_release_staging.json

```diff
@@ -9,8 +9,8 @@
       "orchestratorRelease": "1.17",
```

Comment:
Should we update this while we are making updates?
Same with kubernetes_release_staging_serial.json?

Reply:
Or leave it blank like in kubernetes_containerd_master_serial.json  and a few others?

Reply:
```suggestion
      "orchestratorRelease": "",
```

Sure it get overridden in the prow jobs but that makes it less confusing

Reply:
updated